client/src/index.scss

```diff
@@ -1,14 +1,167 @@
 // Your variable overrides
 $headings-margin-bottom: 1rem;
+$light: #d3e9eb;
 
 @import '~bootstrap/scss/bootstrap';
 
 body {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif, 'Times New Roman', Times, serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  min-height: 100vh;
+  position: relative;
+  padding-bottom: 12rem;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.form-check > label {
+  max-width: 100%;
```

Comment:
Since these properties are the same as `.review-answers`, we can combine them into a comma-separated declaration